src/SarifWeb/Startup.cs

```diff
@@ -36,8 +37,18 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 app.UseHsts();
             }
+
             app.UseHttpsRedirection();
-            app.UseStaticFiles();
+
+            app.UseStaticFiles(new StaticFileOptions
+            {
+                OnPrepareResponse = ctx =>
+                {
+                    const int durationInSeconds = 60 * 60 * 24 * 365;
```

Comment:
>60 * 60 * 24 * 365 [](http://example.com/codeflow?start=50&length=18)

do we really want such long cache period?
if the static files updated in server with same name, client side won't get the last file after 1 year, unless force refresh the browser?